crypto/blst/utils.go

```diff
@@ -0,0 +1,25 @@
+package blst
+
+import (
+	"encoding/binary"
+	"fmt"
+	"math/big"
+)
+
+func ToScalars(coefficients []*big.Int) []*blstScalar {
+	scalars := make([]*blstScalar, 0, len(coefficients))
+	bytes := make([]byte, BlstScalarBytes)
+
+	for _, c := range coefficients {
+		// coefficients that exceed 32 bytes should get filtered before arriving here
+		if c.BitLen() > BlstScalarBytes {
```

Comment:
```suggestion
		if c.BitLen() > BlstScalarBytes * 8 {
```

This is a bug right? Or do you mean to use `QuorumCap` instead of `BlstScalarBytes`?